assets/site.css.liquid

```diff
@@ -702,6 +702,12 @@ a:visited {
   color: inherit;
   text-decoration: none;
 }
+p a,
```

Comment:
well there is a min version. we might as well only use that and not bother tracking this file at all